### PR TITLE
Usability - Default sorting on the admin view all holidays screen - sort by status - show pending first

### PR DESCRIPTION
**Pull request by paulmatchett**
## Relevant Issues

https://trello.com/c/KsmFU7C7/13-usability-default-sorting-on-the-admin-view-all-holidays-screen-sort-by-status-show-pending-first

## Major Changes Made

* Updated list of holidays to be pending first, in date order
* Updated list of pending holidays to be in date order

## AC List

It would be useful if on the /admin/holidays screen it automatically sorted by Status and showed pending holidays first

### CODE REVIEW

**Review by d1wilko — CHANGES_REQUESTED** [review]
Code looks clean but I'm not convinced you have your linter is set up correctly? - can you confirm and then I can approve

File: src/utilities/holidayOrdering.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+      cancelledHolidays.push(holiday);
+    } else {
+      approvedHolidays.push(holiday);
+    }
+  });
+
+  // return holidays in order of pending, approved, rejected, approved
+  return  [...pendingHolidays, ...rejectedHolidays, ...approvedHolidays, ...cancelledHolidays];
+
+};
```
Comment (d1wilko) [inline src/utilities/holidayOrdering.js]: need a new line at the bottom here :)

**Review by d1wilko — CHANGES_REQUESTED** [review]
one change then good to approve :)

File: src/utilities/holidayOrdering.js (hunk 1 above)
Comment (paulmatchett) [reply src/utilities/holidayOrdering.js]: 👍 New line added

**Review by d1wilko — APPROVED** [review]

**Pull request merged**